### PR TITLE
[KOGITO-4546] - Create make olm-manifests target responsible for crea…

### PR DESCRIPTION
**Pull request by Kaitou786**
…ting manifests in same pattern accepted by the OLM

See: https://issues.redhat.com/browse/KOGITO-4546

Signed-off-by: Tarun Khandelwal <tarkhand@redhat.com>

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors' guide](https://github.com/kiegroup/kogito-cloud-operator/blob/master/README.md#contributing-to-the-kogito-operator)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains a link to the JIRA issue
- [x] Pull Request contains a description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster
- [ ] You've added a [RELEASE_NOTES.md](RELEASE_NOTES.md) entry regarding this change

### CODE REVIEW

**Review by ricardozanini — CHANGES_REQUESTED** [review]
Do you mind writing a session in the CONTRIBUTING or README file explaining when we would need this manifest? I assume we would need it to send a PR to the Community Operators project.

File: README.md [hunk 1]
```diff
@@ -447,3 +448,12 @@ Running Kogito operator in remote debug on VSCode and GoLand is very similar to
...
 If you made changes in the core/runtimes part of the Kogito and want to test your changes against the operator. Please follow this [guide](docs/GUIDE_TO_KOGITO_DEVS.md) to test your changes.
+
+## Releasing Kogito Operator
+
+When releasing the new version of kogito-operator in [community-operators](https://github.com/operator-framework/community-operators). One can use `make olm-manifests` it would create the manifests in the `build/_output/olm/<version>` directory in format which the community-operators repo expects.
+One can then just copy the directory into the [kogito-operator](https://github.com/operator-framework/community-operators/tree/master/community-operators/kogito-operator) and raise the PR.
+
+Note: One needs to create two separate PRs with this folder added in [upstream-kogito-operator](https://github.com/operator-framework/community-operators/tree/master/upstream-community-operators/kogito-operator) and [community-kogito-operator](https://github.com/operator-framework/community-operators/tree/master/community-operators/kogito-operator) respectively.
+
+Before raising the PR, make sure the `replaces` field in the CSV is correct
```
Comment (ricardozanini) [inline README.md:459]:
```suggestion
Before raising the PR, make sure the `replaces` field in the CSV is correct.
```

**Review by ricardozanini — APPROVED** [review]

**Review by ricardozanini — APPROVED** [review]

**Review by sutaakar — APPROVED** [review]

**Comment by Kaitou786** [comment]
Thinking about it, do you guys think we should cherry-pick to the 1.3.x branch as well?
Would be easier when doing the release

**Comment by Kaitou786** [comment]
Removed the metrics files as they are causing problems when upgrading the operator @ricardozanini many thanks for the suggestion! <3

**Review by sutaakar — APPROVED** [review]

**Review by ricardozanini — APPROVED** [review]

**Review by spolti — APPROVED** [review]

**Pull request merged**